source/conditionally.ts

```diff
@@ -0,0 +1,35 @@
+declare const webpHero;
+
+/**
+ * Check if WebP is supported by the current browser.
+ */
+ export function isWebpSupported(callback) {
+	var img = new Image();
+	img.onload = function () {
+		var result = (img.width > 0) && (img.height > 0);
+		callback(result);
+	};
+	img.onerror = function () {
+		callback(false) ;
+	};
+	img.src = "data:image/webp;base64,UklGRiIAAABXRUJQVlA4IBYAAAAwAQCdASoBAAEADsD+JaQAA3AAAAAA"; // Lossy WebP image.
+}
+
+(function() {
+	isWebpSupported(function(support) {
+		if (! support) {
+			var js = document.createElement('script');
+			js.src = 'https://unpkg.com/webp-hero@0.0.2/dist-cjs/polyfills.js';
```

Comment:
i still don't understand what the polyfills do exactly vs the bundle :(, is that documented somewhere? So are you saying we can leave off the polyfill loading here entirely?

Reply:
@adamsilverstein sorry about the late reply,

yes, we should probably not engage in loading any polyfills via the `conditionally` script.

- the [polyfills](https://github.com/chase-moskal/webp-hero/blob/master/package.json#L22) include one for `Promise`, and another for `Array`
- the inclusion of polyfills should be deferred to the author of the html page, who may decide to include their own polyfills for the other parts of their application, so we want to leave it in their hands (to avoid double-loading any polyfills)